app/jobs/bulk_update/analytics_job.rb

```diff
@@ -3,6 +3,8 @@
 module BulkUpdate
   class AnalyticsJob < ApplicationJob
     def perform(model:, ids:)
+      return unless FeatureService.enabled?("google.send_data_to_big_query")
```

Comment:
Do you think we should just use the DfE::Analytics API here:
``` Ruby
  DfE::Analytics.enabled?
```

Because it will be set to the Feature flag in anycase and we should I reckon just confine how it's been set to the analytics config.

Reply:
Fixed